wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/ProjectService.java

```diff
@@ -351,8 +361,16 @@ public void importProject(@ApiParam(value = "Path in the project", required = tr
                                                                      ServerException,
                                                                      NotFoundException,
                                                                      BadRequestException {
-        projectManager.importProject(path, sourceStorage, force,
-                                     () -> new ProjectImportOutputWSLineConsumer(path, workspace, 300));
+
```

Comment:
The main task was just streaming importer logs over JSON-RPC protocol. @vparfonov and I had discussion about this moment before and we decided to make only logs streaming at first step. Running import operation over JSON-RPC will be next step of this global task.